pysrc/scripts/evaluation.py

```diff
@@ -39,72 +42,167 @@ def create_evaluation_csv(runs: MetanomeRunBatch, output_file: str, config: Glob
     return output_csv
 
 
-# TODO: Actually implement this
-def make_plots(output_file: str, plot_folder: str, config: GlobalConfiguration) -> str:
-    df = pd.read_csv(os.path.join(os.getcwd(), config.output_folder, output_file + '.csv'))   
-    
-    f, (ax1, ax2) = plt.subplots(1, 2, figsize=(10, 15))
-    sns.despine(f)
+def create_TpFpFn_stacked_barplot(axes : matplotlib.axes.Axes, dataframe: pd.DataFrame, groupby_attr: str) -> matplotlib.axes.Axes:
```

Comment:
Consider extracting those methods to a separate file or even one file per method (in a new subfolder).